lib/runner/concurrency/index.js

```diff
@@ -77,10 +166,10 @@ class Concurrency extends EventEmitter {
         jobs.push(...jobList);
       }
 
-      return this.runMultipleTestWorkers(jobs, args, availColors);
+      return this.runMultipleTestWorkers(jobs, availColors);
     }
 
-    return this.runTestEnvironments(envs, args, availColors);
+    return this.runWorkerTestEnvironments(envs, availColors);
```

Comment:
We don't need this flow, see discussion here: https://github.com/nightwatchjs/nightwatch/pull/3590#issuecomment-1450242970

Reply:
Let's tackle that in another PR. Don't want to clutter this PR with multiple changes.

Reply:
okay